utils/profile_tool/profile.py

```diff
@@ -43,14 +44,20 @@ def __init__(self, path, title):
         self.path = path
         self.title = title
         self.rules = []
+        self.variables = []
```

Comment:
I'd say it's better to use a dictionary where the key-value pair is the name and value of the variable. This will improve the lookup time of the variable and also simplify the code to find the value of the variable.

Reply:
I am using this in a context that I don't know the variable name. But let me check here how I can change the behavior in my case.

Reply:
I simplified it in the last commit.

Reply:
Hum, I see now what you meant. You are right. And I think it would be even easier to work with them this way. Thanks. Done in the last commit.